src/Domains/Registrar/Adapter/Mock.php

```
@@ -508,6 +481,31 @@ public function checkTransferStatus(string $domain, bool $checkStatus = true, bo
         }
     }
 
+    /**
+     * Update the nameservers for a domain
+     *
+     * @param string $domain
+     * @param array $nameservers
+     * @return array
+     */
+    public function updateNameservers(string $domain, array $nameservers): array
```

PHPMD (2.15.0)

Avoid unused parameters such as '$domain'. (undefined)

(UnusedFormalParameter)